detect_secrets/plugins/keyword.py

```diff
@@ -80,53 +81,81 @@
     "\\k.*'",
     '`cat',
     '`grep',
+    '`sudo',
     'account_password',
+    'api_key',
```

Comment:
Why is this in both `DENYLIST` and `FALSE_POSITIVES`?

Reply:
So this is for the case wherein silly code does `magical_api_key: api_key`, or something to that effect, normally in a test.

`DENYLIST` will catch the fact that it could be an API key from `magical_api_key` on the LHS, but then `FALSE_POSITIVES` will check the RHS and cause us not to alert on it.